examples/run-blockchain/test-data.json

```diff
@@ -0,0 +1,159 @@
+{
+  "blocks": [
+    {
+      "blockHeader": {
+        "bloom": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+        "coinbase": "8888f1f195afa192cfee860698584c030f4c9db1",
+        "difficulty": "0x020000",
+        "extraData": "0x",
+        "gasLimit": "0x023ec6",
+        "gasUsed": "0x021536",
+        "hash": "f53f268d23a71e85c7d6d83a9504298712b84c1a2ba220441c86eeda0bf0b6e3",
+        "mixHash": "29f07836e4e59229b3a065913afc27702642c683bba689910b2b2fd45db310d3",
+        "nonce": "8957e6d004a31802",
+        "number": "0x01",
```

Comment:
oh! so all that homestead stuff i said was good to remove... well this is why we have it. The block number is set to 1. The homestead is detected on the block number which  is much higher, yet the test expect the VM to run in homestead mode! Unfortaintly you can just change the number either, b/c it will invalidated the PoW. On option for the example could be just to change the number and don't check do the the PoW check